docs/stats.md

```diff
@@ -0,0 +1,38 @@
+# Stats
+
+## Overview
+<!-- TODO: move module definitions here? -->
```

Comment:
Ya I like this, we can replace the module definitions in the files themselves w/ links to the corresponding doc file